test/Unit/DependencyManagement/DependencyManagerTests.cs

```diff
@@ -50,6 +50,38 @@ public void Initialize_ReturnsNewSnapshotPath_WhenNoAcceptableDependencyVersions
             }
         }
 
+        [Fact]
+        public void Initialize_Throws_WhenDependenciesAreDefinedInRequirementsPsd1_OnLegion()
```

Comment:
Consider adding a test that verifies that we _don't_ throw an exception on Legion when the dependency manifest is empty

Reply:
Test case added.